ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dhcp-servers.yml

```diff
@@ -12,6 +12,14 @@ dhcp_servers:
           - code: 66
```

Comment:
add one more test with only code to improve the coverage

Reply:
Fixed